### PR TITLE
Next generation

### PR DESCRIPTION
**Pull request by stefanpearson**
See [Vizia Facelift document](https://docs.google.com/document/d/1wqvGWyRAvqPP63o7h0MNI4H8Rnv0HGJ_KwHdf1k1j1I).

* Backwards compatible, until some of these things can be phased out (mostly the colours). I have removed properties / classes that are definitely not used, after doing some digging across the whole org's source code. All remaining custom properties are a shim / alias to the new design tokens.
* Moves legacy CSS custom properties and classes to a `compat` directory.
* Builds CSS from [Vizia Design Tokens](https://github.com/vizia/design-tokens).

I've tested this with every destination in the monorepo, and the only thing that needs to be amended is the Highcharts root font size (easily fixable).

Theme building will need to be synced with this, so rollout will have to happen at the same time.

### CODE REVIEW

File: src/css/global.css [hunk 1]
```diff
@@ -1,6 +1,7 @@
 :root {
     /* Root font-size = 10px / 0.1rem = 1px */
     font-size: 62.5%;
+    font-size-adjust: 100%;
```
Comment (lcapel) [inline src/css/global.css]: What's the point of setting this if you don't ever change it?

File: src/css/global.css [hunk 2]
```diff
@@ -11,12 +12,12 @@
...
 body {
-    user-select: none;
     font-family: var(--viz-type-face-base);
     font-weight: var(--viz-type-weight-normal);
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     font-smoothing: always;
     text-rendering: optimizeLegibility;
     text-decoration-skip-ink: auto;
+    user-select: none;
```
Comment (lcapel) [inline src/css/global.css]: Y not tho?

File: scripts/build.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        [
+            path.resolve('src/css/global.css'),
+            // Backwards compatibility
+            path.resolve('src/css/compat/colors.css'),
+            // Backwards compatibility
+            path.resolve('src/css/compat/icons.css'),
+            // Backwards compatibility
+            path.resolve('src/css/compat/layout.css')
+        ]
+            .map((path) => fs.readFile(path))
```
Comment (lcapel) [inline scripts/build.js]: what it this disgusting formatting? 😆

File: src/css/global.css (hunk 2 above)
Comment (stefanpearson) [reply src/css/global.css]: Vizia's made to feel like a native app, not a website, you can't highlight in apps. It's an edge case but I've seen a mouse accidentally jogged on the dogfooding screens, which ended up with a gnarly highlight over everything

File: src/css/global.css (hunk 1 above)
Comment (stefanpearson) [reply src/css/global.css]: Some browsers have a shitty default that will change the x-height of fonts to make them much bigger, I think iOS does it (or used to). This just makes sure nothing weird happens.

**Pull request merged**